### PR TITLE
Extend metadata creation pages (Project/Dataset/Distribution)

### PR DESCRIPTION
**Pull request by Kritzzz**
See Issue [aegisbigdata/documentation#131](https://github.com/aegisbigdata/documentation/issues/131).

### CODE REVIEW

File: aegis-web/yo/app/index.html [hunk 1]
```diff
@@ -68,7 +68,7 @@
     <link rel="stylesheet" href="bower_components/angularjs-slider/dist/rzslider.css" />
     <link rel="stylesheet" href="bower_components/nvd3/build/nv.d3.css" />
     <link rel="stylesheet" href="bower_components/angular-bootstrap-toggle/dist/angular-bootstrap-toggle.min.css" />
-    <link rel="stylesheet" href="bower_components/css-hamburgers/dist/hamburgers.css" />
```
Comment (cbotsikas) [inline aegis-web/yo/app/index.html]: Why is this removed? Please check all references.

File: aegis-web/yo/bower.json [hunk 2]
```diff
@@ -51,7 +51,11 @@
     "angular-file-saver": "^1.1.3",
     "angular-google-chart": "^0.1.0",
     "apexcharts": "npm:apexcharts#^2.5.0",
-    "css-hamburgers": "^1.1.3"
```
Comment (cbotsikas) [inline aegis-web/yo/bower.json:54]: Same here

**Review by cbotsikas — CHANGES_REQUESTED** [review]
See inline comments

**Comment by cbotsikas** [comment]
I think this is a common issue, seen in other PRs too. I haven't looked for the reason but some references are automatically changed at some point. for example on older PR #31 see[ the revert commit](https://github.com/aegisbigdata/hopsworks/pull/31/commits/790e738b623a29481ad3bbb590f7ac6c244639d7)

Please check what changed you need in index.html and revert the rest. Same for bower.json

File: aegis-web/yo/bower.json (hunk 2 above)
Comment (cbotsikas) [reply aegis-web/yo/bower.json:54]: ah, this one is ok. I didn't notice the reference below.

**Comment by cbotsikas** [comment]
bower.json is fine. Just check index.html

**Review by fabiankirstein — APPROVED** [review]

**Comment by fabiankirstein** [comment]
@cbotsikas I fixed it. I think those includes are generated by Grunt. So it actually includes what it needs... We should ask KTH.

**Review by cbotsikas — APPROVED** [review]

**Pull request merged**